src/pages/en/guides/cms/wordpress.md

```diff
@@ -2,12 +2,222 @@
 title: Headless WordPress & Astro
 description: Add content to your Astro project using WordPress as a CMS
 layout: ~/layouts/CMSLayout.astro
-stub: true
+stub: false
 service: WordPress
 ---
 
 [WordPress](https://wordpress.org/) is a content management system that includes its own frontend, but can also be used as a headless CMS to provide content to your Astro project.
 
+## Integrating with Astro
+
+WordPress comes with a built-in [WordPress REST API](https://developer.wordpress.org/rest-api/) to connect your WordPress data to Astro. You can optionally install [WPGraphQL](https://wordpress.org/plugins/wp-graphql/) on your site to use GraphQL.
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **A WordPress site** - Your site's REST API is `[YOUR_SITE]/wp-json/wp/v2/` and is available by default with any WordPress site. It is also possible to [set up WordPress on a local environment](https://wordpress.org/support/article/installing-wordpress-on-your-own-computer/).
+
+### Setting up Credentials
+
+Your WordPress REST API is available to external requests for data fetching of your posts without authentication by default. This does not allow users to modify your data or site settings and allows you to use your data in your Astro project without any credentials.
+
+You may choose to [require authentication](https://developer.wordpress.org/rest-api/frequently-asked-questions/#require-authentication-for-all-requests) if necessary.
+
+### Installing Depenencies
+
+There are no dependencies required to fetch your WordPress data from Astro via the REST API.
+
+### Fetching Data
+
+You will fetch your WordPress data through your site's unique REST API URL and the route for your content. (For a blog, this will commonly be `posts`.) Then, you can render your post's properties using Astro's `set:html=""` directive. For example, to display a list of post titles and their content:
+
+```astro title="src/pages/index.html
+---
+import Layout from '../layouts/Layout.astro';
+const response = await fetch("https://[YOUR-SITE]/wp-json/wp/v2/posts")
+const allPosts = await response.json()
+---
+<Layout>
+<h1>My Posts</h1>
+{
+  allPosts.map((post) => (
+    <>
+      <h2 set:html="post.title.rendered" />
+      <p set:html="post.content.rendered" />
+    </>
+  ))
+}
+</Layout>
+```
+
+The WordPress REST API includes [global parameters](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/) such as `_fields` and `embed`. 
+
+A large quantity of data is available to you via this API, so you may wish to only fetch certain fields. You can restrict your response by adding the `_fields` paramater to the API URL, for example: `[YOUR-SITE]/wp/v2/posts?_fields=author,id,excerpt,title,link` 
```

Comment:
Two thoughts here:

1. We link `_embed` below but not `_fields` here. I'm ok with either decision, but we should probably be consistent. Note that I hadn't initially linked because I did link "global parameters" which is where these, and more, can be found. So let's make a decison which way to go!
2. In the WordPress docs for `_fields`, they do state two acceptable ways of writing these fields. I chose one in the original, and Kevin changed it to the other.  😅  Just wanted to make sure this was an intentional decision that you prefer the other syntax (I thought the one where you just chain onto the URL seemed clean, and easy to understand?)  I don't think we *need* to show both, but I'm not opposed to doing so if people prefer it. Do we think one is better than the other?